ci-operator/step-registry/ipi/conf/ibmcloud/default/ipi-conf-ibmcloud-default-commands.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+REGION="${LEASED_RESOURCE}"
+echo "Azure region: ${REGION}"
+
+workers=${WORKERS:-3}
+if [ "${WORKERS}" -le 0 ]; then
+  workers=0
+fi
+ocp_arch="amd64"
+PATCH="${SHARED_DIR}/install-config-common.yaml.patch"
+cat > "${PATCH}" << EOF
+baseDomain: ${BASE_DOMAIN}
+platform:
+  ibmcloud:
+    region: ${REGION}
+controlPlane:
+  architecture: ${ocp_arch}
```

Comment:
If only "adm64" is supported for IBM cloud, I think we can just omit the `architecture` parameter